src/components/producerItems/components/ProductEditModal.js

```diff
@@ -93,41 +99,96 @@ class ProductEditForm extends React.Component {
         const { form, productToEdit, updateProduct } = this.props;
         const { resetFields, validateFields } = form;
 
-        validateFields((error, fieldValues) => {
-            if (!error) {
-                updateProduct({ ...productToEdit, ...fieldValues });
-                return resetFields();
-            }
-
-            return error;
-        });
-    }
-
-    static getDerivedStateFromProps(props) {
-        const { productToEdit, form: { getFieldDecorator } } = props;
-        const formItems = generateProductEditForm(getFieldDecorator, productToEdit);
-
-        return { formItems };
-    }
-
-    render() {
-        const { closeModal, isNewProduct, isOpen, isProductUpdating } = this.props;
-        const { formItems } = this.state;
-        const title = isNewProduct ? CREATE_TITLE : EDIT_TITLE;
-
-        return (
-            <Modal
-                confirmLoading={isProductUpdating}
-                visible={isOpen}
-                title={title}
-                okText={SAVE_PRODUCT_DETAILS}
-                onCancel={closeModal}
-                onOk={this.updateProductInfo}
-            >
-                <Form>{formItems}</Form>
-            </Modal>
-        );
+        if(this.state.fileDetails.length>=1){
+            const { image_url, image_id } = this.state.fileDetails[0].response.data;
```

Comment:
given that you are access `this.state.fileDetails` more than once, you should destructure it.
```
const { fileDetails } = this.state;
```